src/lib/tray.ts

```diff
@@ -28,48 +31,72 @@ export const startFlowTimer = async (startTime: DateTime) => {
   }
 
   if (!tray) return
-  await tray.setIcon(null)
-  
-  let currentDuration = useFlowTimer.getState().duration
-  
-  const unsubscribe = useFlowTimer.subscribe(
-    (state) => {
-      currentDuration = state.duration
-    }
-  )
+  await tray.setTitle('')
   
   timerInterval = setInterval(async () => {
-    let timerDuration: Duration
-    if(!currentDuration) {
-      timerDuration = startTime.diffNow().negate()
+    const currentTotalDuration = useFlowTimer.getState().totalDuration
+
+    let formattedTime: string
+    let elapsedMs: number
+    let totalMs: number
+    let shouldStop = false
+
+    if (currentTotalDuration) {
```

Comment:
This is a bit hard to read. Two things I would do

- create a small function `getRemainingDuration`
- combine the two if statements that check remaining duration into one if statement.